tests/profile/test_profile.py

```diff
@@ -233,26 +234,18 @@ def test_BasicDatasetProfiler_on_titanic():
     and comparing the EVRs to ones retrieved from a
     previously stored file.
     """
-    df = ge.read_csv("./tests/test_sets/Titanic.csv")
+    df = ge.read_csv(file_relative_path(__file__, "../test_sets/Titanic.csv"))
     suite, evrs = df.profile(BasicDatasetProfiler)
 
     # Check to make sure BasicDatasetProfiler is adding meta.columns with a single "description" field for each column
-    # print(json.dumps(suite.meta, indent=2))
     assert "columns" in suite.meta
     for k, v in suite.meta["columns"].items():
         assert v == {"description": ""}
 
     # Note: the above already produces an EVR; rerunning isn't strictly necessary just for EVRs
     evrs = df.validate(result_format="SUMMARY")
 
-    # with open('tests/test_sets/expected_evrs_BasicDatasetProfiler_on_titanic.json', 'w+') as file:
-    #     json.dump(expectationSuiteValidationResultSchema.dump(evrs).data, file, indent=2)
-    #
-    # with open('tests/render/fixtures/BasicDatasetProfiler_evrs.json', 'w+') as file:
-    #     json.dump(expectationSuiteValidationResultSchema.dump(evrs).data, file, indent=2)
```

Comment:
@eugmandel IIRC, these lines are uncommented when this test is snapshotted, correct? If so, we should add a comment to that effect rather than removing them here.

Reply:
@jcampbell correct

Reply:
K. I'll add a comment and revert this then merge this branchy branch.